packages/shared/src/types/targeting/shared.ts

```diff
@@ -37,7 +37,11 @@ export interface PurchaseInfo {
 
 export const abandonedBasketSchema = z.object({
     amount: z.union([z.number(), z.literal('other')]),
-    billingPeriod: z.string(),
+    billingPeriod: z.union([
```

Comment:
an alternative way to do this would be like this: [with z.enum](https://github.com/guardian/support-dotcom-components/blob/61ca5815f3fef0cec879d9118c6e44476c7bbce4/packages/shared/src/types/abTests/shared.ts#L15)

Reply:
with zod it would like like

z.enum(['a', 'b', 'c']),